package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@viamrobotics/sdk",
+  "version": "0.0.1",
+  "description": "",
+  "main": "./dist/main.umd.js",
+  "module": "./dist/main.es.js",
```

Comment:
did you verify this works well with rdk/npm start?

Reply:
yes